packages/api/modules/users/procedures/index.ts

```diff
@@ -0,0 +1,11 @@
+import { getProfileRouter } from './get-profile';
+import { updateProfileRouter } from './update-profile';
+import { searchUsersRouter } from './search-users';
+
+import { router } from '../../../trpc/base';
+
+export const usersRouter = router({
+  ...getProfileRouter._def.record,
+  ...updateProfileRouter._def.record,
+  ...searchUsersRouter._def.record,
+});
```

Comment:
nitpick: we usually here just export the procedure

```suggestion
export { getProfileRouter } from './get-profile';
export { updateProfileRouter } from './update-profile';
export { searchUsersRouter } from './search-users';
```